test/runtests.jl

```diff
@@ -1,6 +1,7 @@
 using Base.Broadcast: broadcastable
 using ChainRules
 using ChainRulesCore
+using ChainRulesCore: NoTangent, rrule
```

Comment:
```suggestion
```
These should be exported and just work because of the line above

Reply:
Cool - I've removed the commit that added the tests (I was half expecting to do this anyway). 

Now the changeset is just the Project.toml and adding the no diff rule.